### PR TITLE
handle spaces in the absolute path (#1845)

### PR DESCRIPTION
**Pull request by eoola**
# Motivation & context
When the developer's parent directory had spaces in them, it caused the [absolute path scenario](https://github.com/cucumber/cucumber-js/issues/1845) in the formatter paths feature to fail. 

We discovered the scenario didn't work because it required quotations around the path to be read into the CLI properly since it had spaces in it. Otherwise, it would split the path into separate arguments as it's supposed to. We added quotes to the scenario to ensure the path was read as one argument but then we discovered that the underlying [cli index code](https://github.com/cucumber/cucumber-js/blob/fd540c048a30a74328ed509b3f6b2ca1fe991f93/src/cli/index.ts#L115) couldn't cope with the path with quotes around them. 

We changed the configuration builder to clean up the quotes in the path before passing to the cli.

## Type of change

- Refactoring/debt (improvement to code design or tooling without changing behaviour)

# Checklist:

- [ ] I have added an entry to the "Unreleased" section of the [**CHANGELOG**](../CHANGELOG.md), linking to this pull request.

### CODE REVIEW

File: features/formatter_paths.feature [hunk 1]
```diff
@@ -3,15 +3,15 @@ Feature: Formatter Paths
   Background:
     Given a file named "features/a.feature" with:
       """
-      Feature: some feature
-        Scenario: some scenario
-          Given a passing step
+Feature: some feature
```
Comment (mattwynne) [inline features/formatter_paths.feature:6]: Ooops @eoola we shouldn't have pushed these reformatting changes.

Comment (mattwynne) [reply features/formatter_paths.feature:6]: It was probably your editor doing an autoformat on the file I guess.

Can you see if you can push another commit to the branch that puts the indentation of these lines back to what it was before?

**Comment by mattwynne** [comment]
Woop, we fixed it!

We just need to add a unit test for the change to `ConfigurationBuilder`

**Comment by aurelien-reeves** [comment]
Would it be possible to have some explanation regarding the issue you faced, and the fix that is implemented?
The issue mention an issue with some spaces, but the fix seems related to quotes, so I am a confused here 😅

**Comment by eoola** [comment]
> Would it be possible to have some explanation regarding the issue you faced, and the fix that is implemented?
> The issue mention an issue with some spaces, but the fix seems related to quotes, so I am a confused here 😅

@aurelien-reeves We've updated the description to properly reflect the problem as well as the suggested fix. Is that sufficient? Or do you recommend any more changes to the description?

**Review by aurelien-reeves — APPROVED** [review]

**Comment by mattwynne** [comment]
Ooh @eoola we need to update the CHANGELOG.md before we merge this - can you try and do that yourself and push the change this branch?

File: features/formatter_paths.feature [hunk 2]
```diff
@@ -23,9 +23,10 @@ Feature: Formatter Paths
       <duration-stat>
       """
 
+  @wip
```
Comment (davidjgoss) [inline features/formatter_paths.feature:26]: Don’t forget to remove this tag before merging

Comment (eoola) [reply features/formatter_paths.feature:26]: will do!

**Comment by eoola** [comment]
> Ooh @eoola we need to update the [`CHANGELOG.md`](https://github.com/cucumber/cucumber-js/blob/main/CHANGELOG.md) before we merge this - can you try and do that yourself and push the change this branch?

@mattwynne Anything you'd like me to add to that?

**Comment by aurelien-reeves** [comment]
> Ooh @eoola we need to update the [`CHANGELOG.md`](https://github.com/cucumber/cucumber-js/blob/main/CHANGELOG.md) before we merge this - can you try and do that yourself and push the change this branch?

> Don’t forget to remove this tag before merging

I may have review it too quickly 😓

**Comment by mattwynne** [comment]
> > Ooh @eoola we need to update the [`CHANGELOG.md`](https://github.com/cucumber/cucumber-js/blob/main/CHANGELOG.md) before we merge this - can you try and do that yourself and push the change this branch?
> 
> @mattwynne Anything you'd like me to add to that?

Yes, so if you have a look at that file, you need to add a new bullet point in the "Unreleased" section that looks like the other ones, but links to this ticket.

**Comment by eoola** [comment]
> > > Ooh @eoola we need to update the [`CHANGELOG.md`](https://github.com/cucumber/cucumber-js/blob/main/CHANGELOG.md) before we merge this - can you try and do that yourself and push the change this branch?
> > 
> > 
> > @mattwynne Anything you'd like me to add to that?
> 
> Yes, so if you have a look at that file, you need to add a new bullet point in the "Unreleased" section that looks like the other ones, but links to this ticket.
> 
> I would put it under "Fixed", something like:
> 
> ```gfm
> ### Fixed
> - Handle spaces in the absolute path ([#1845](https://github.com/cucumber/cucumber-js/pulls/1845))
> ...
> ```
> 
> Does that make sense? It's not a big deal, but it's a good habit to put these changes in as part of the PR, otherwise someone has to remember to do it after the PR has been merged.

Done. Those changes should be shown in a second

File: features/formatter_paths.feature (hunk 2 above)
Comment (eoola) [reply features/formatter_paths.feature]: done

**Review by davidjgoss — APPROVED** [review]

**Comment by mattwynne** [comment]
🎉 

That's your 🏆 second PR merged @eoola! 🏆

**Pull request merged**